test/s3select_test.cpp

```diff
@@ -1579,7 +1579,65 @@ TEST(TestS3selectFunctions, coalescewhere)
     ASSERT_EQ(s3select_result, std::string("true,\n"));
 } 
 
+TEST(TestS3selectFunctions, castint)
+{
+    s3select s3select_syntax;
+    const std::string input_query = "select cast(5.123 as int) from stdin ;" ;
```

Comment:
its constant test , as by itself it OK.
it need for more complex unit-test.
cast (complex-expression-return-some-number-float-or-int  as int)
and also negative test 
string as int
timestamp as int